lib/ansible/modules/packaging/language/pip.py

```diff
@@ -190,6 +190,34 @@
   become: True
 '''
 
+RETURN = '''
+cmd:
+  description: pip command used by the module
+  returned: success
+  type: string
+  sample: pip2 install ansible six
```

Comment:
umm... This doesn't seem like a majority use case of this option because ideally people wouldn't put in ansible and six as part of this option but they'd put it in name. How about "pip2 install --require-hashes"

Reply:
@SurenNihalani that's what would show if you had:
```
- name: install things
  vars:
    packages:
      - ansible
      - six
  pip:
    name: "{{ packages }}"
    state: present
```

This is an example and that's unfortunately what I came up with on the spot.
Do you have a better idea for example modules ?

Reply:
Hmm... I didn't understand the documentation correctly then. I thought this was the command that would be used before formatting all the arguments in (similar to tox's `install_command`). This seems ideal.